priv/repo/dummy.exs

```diff
@@ -0,0 +1,11 @@
+# Script for populating the database with disposable, development-oriented data. You can run it as:
+#
+#     mix run priv/repo/dummy.exs
+#
+# Inside the script, you can read and write to any of your
+# repositories directly:
+#
+#     ElixirBoilerplate.Repo.insert!(%ElixirBoilerplate.SomeSchema{})
```

Comment:
Should we add a note stating that we recommend the usage of factories with `ex_machina`? I don’t recall any dummy data using `insert!` nor `update!`.

Reply:
Yes, good idea. Done!